daml-script/daml/Daml/Script.daml

```diff
@@ -734,6 +789,13 @@ fromAnyContractId cid
   | cid.templateId == templateTypeRep @t = Some (coerceContractId cid.contractId)
   | otherwise = None
 
+toAnyContractId : forall t. Template t => ContractId t -> AnyContractId
+toAnyContractId cid = AnyContractId
+  { templateId = templateTypeRep @t
+  , contractId = coerceContractId cid
+  }
```

Comment:
I noticed we have an almost identical function (with constraint `TemplateOrInterface t` instead of `Template t`) in `daml-trigger`, perhaps we can merge them together in `daml-stdlib` - but that's definitely for another PR